db/migrate/20200317162403_remove_disability_compensation_job_statuses.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveDisabilityCompensationJobStatuses < ActiveRecord::Migration[5.2]
+  def change
+    drop_table :disability_compensation_job_statuses
```

Comment:
I don't really like using `change` here because the automagic `down` would be to create the table.  This doesn't actually reverse the effects of dropping a table because creating the table doesn't reinstate dropped data, dropping a table is in reality irreversible. So, I originally _wanted_ to write`up/down` functions instead and have my `down` `raise ActiveRecord::IrreversibleMigration` but that kind of stinks too because there isn't any data in this table or the `disability_compensation_submissions` table in prod-- it seems silly to fail rollbacks for this. In summation, all these solutions are imperfect, so I picked the least controversial way forward.